src/networking.c

```diff
@@ -1893,6 +1897,9 @@ void processInputBuffer(client *c) {
 }
 
 void readQueryFromClient(connection *conn) {
+    // Update total number of reads on server
+    server.stat_total_reads_processed++;
```

Comment:
I think we should move it after `postponeClientRead`, cause after that the read event is executed, @oranagra @madolson